core/conversion/var/Var.cpp

```diff
@@ -85,6 +85,27 @@ std::string Var::type_name() const {
   }
 }
 
+nvinfer1::ITensor* Var::ITensorOrFreeze(ConversionCtx* ctx, const torch::jit::Node* n) {
+  TRTORCH_CHECK(isITensor() || isIValue(), "Requested either IValue or ITensor, however Var type is " << type_name());
```

Comment:
The check should be `isITensor() || isIValue() && IValue().isTensor()`. It either needs to be a ITensor or a IValue that contains a Tensor. Or are you intending to use this in the scalar case too?

Reply:
No, you're correct. I'll add it in. Thanks!